packages/main/src/Tokenizer.ts

```diff
@@ -550,19 +569,19 @@ class Tokenizer extends UI5Element {
 	}
 
 	get showNMore() {
-		return !this.expanded && this.showMore && this.overflownTokens.length;
+		return !!(!this.expanded && this.showMore && this.overflownTokens.length);
```

Comment:
Better to set the double negation individually to all parts of the expression that must be forced to boolean (probably just the third as far as I can see)

Reply:
Done